packages/components/Menu/src/MenuPopover/MenuPopover.tsx

```diff
@@ -47,11 +64,49 @@ export const MenuPopover = compressible<MenuPopoverProps, MenuPopoverTokens>(
             }
           : state.innerView;
       const content = React.createElement(View, innerViewProps, children);
-      return <Callout {...mergedProps}>{content}</Callout>;
+
+      return Platform.OS === 'android' ? (
```

Comment:
Should we extract this code from MenuPopover.tsx? Another thought I have is can this be stood up as Callout for Android/mobile?

Reply:
Yeah, I think this should either be added as Callout for Android or we should introduce another component which wraps Callout for desktop and the Modal for Android. We should fork files for platforms especially if the majority of the file is going to be platform specific code.

Reply:
I think the later on will be suited "we should introduce another component which wraps Callout for desktop and the Modal for Android" as on Android we don't have design guidelines for it right now. So having a separate component for it would be possible i think. 
@rurikoaraki @chiuam , will it be feasible ?

Reply:
Yeah I'm fine with it being another component and it can be local to Menu for the time being. I would prefer to see that separation done as part of this change because otherwise it's easy to miss the non-Android code in this file.

Reply:
Forked into a different component: MenuCallout. 

Verified on win32, working as before.